lib/cube_browser/__init__.py

```diff
@@ -387,6 +387,11 @@ def __call__(self, **coord_values):
 
         """
         cube = self._get_slice(coord_values)
+        # guess bounds, if required.
+        for cname in self.coords:
+            coord = cube.coord(cname)
+            if len(coord.points) > 1 and not coord.has_bounds():
+                coord.guess_bounds()
```

Comment:
i think it is reasonable for the cube browser to do this;  In fact, even without this code, iris will run the guess_bounds function, it just spits out warnings telling the user it is doing so.

I don't want warnings in the cube browser if we can avoid them